src/main/java/teammates/common/util/Const.java

```diff
@@ -318,8 +318,8 @@ public static final String getjQueryUiFilePath(boolean isDevServer) {
                 "Select this option to have the feedback session become visible "
                 + "when it is open for submissions (as selected above).";
         public static final String FEEDBACK_SESSION_SESSIONVISIBLENEVER =
-                "Select this option if you want the feedback session to never be visible. "
-                + "Use this option if you want to use this as a private feedback session.";
+                "Select this option if you want the feedback session to be private. "
+                + "A private session is never visible to anyone.";
```

Comment:
Can add the following sentence to the end. `Private sessions can be used to record your own comments about others, for your own reference.`